### PR TITLE
add lower versions to REQUIRE

### PR DESCRIPTION
**Pull request by Evizero**
- Colors and ColorVectorSpace I copied the values that Images requires (just in case).
- Interpolations I based it on a recent change you did that concerned BSplines.
- ImageCore seems reasonable

I am a bit lost what to look for at OffsetArrays and StaticArrays. Since Images.jl doesn't list any lower version I just left them as is

### CODE REVIEW

**Comment by Evizero** [comment]
Tests wont run until `recenter` is available at CoordinateTransforms, which is not yet tagged. I made a request there: https://github.com/FugroRoames/CoordinateTransformations.jl/issues/24

**Comment by Evizero** [comment]
We could comment out the `warp` related tests for now and focus on the `restrict` and `imresize` move (?)

**Comment by timholy** [comment]
I'd be fine with you commenting those out, if that's what you prefer.

**Comment by Evizero** [comment]
How should we go about 0.6 testing? The error thrown in appveyor on nightly seems to be an upstream one in Interpolations, which in turn looks like it was fixed on master but not yet tagged

**Comment by timholy** [comment]
I'll tag a new Interpolations release.

File: REQUIRE [hunk 1]
```diff
@@ -1,8 +1,8 @@
 julia 0.5
-ImageCore
+ImageCore 0.1.0
 CoordinateTransformations
```
Comment (andyferris) [inline REQUIRE]: will be 0.3.3 for recenter

Comment (andyferris) [inline REQUIRE]: Sorry v0.4.0 now. https://github.com/JuliaLang/METADATA.jl/pull/7964

**Comment by Evizero** [comment]
I'll go with this for now. I'll reactivate and bump CoordinateTransformations later.

**Pull request merged**